multiple-instances/rust/src/main.rs

```diff
@@ -52,6 +54,53 @@ async fn get_sdk(
     Ok(sdk)
 }
 
+/// On first run (if you don't already have a node), leave `mnemonic` as None. A new mnemonic is
+/// generated and printed to console.
+///
+/// On subsequent runs, or if you already have a node, set the `mnemonic`.
+///
+/// The device key `client-key.pem` and certificate `client.crt` are read from the current directory.
+///
+/// If you have an invite code instead of a partner certificate, use [get_sdk_with_invite_code].
```

Comment:
Can the partner credentials be left empty too on subsequent runs, just like the invite code?